services/reference/linea/json-rpc-methods/eth_call.mdx

```diff
@@ -1,40 +1,13 @@
 ---
 title: "eth_call"
+hide_title: true
+hide_table_of_contents: true
 ---
 
-import Tabs from '@theme/Tabs';
-import TabItem from '@theme/TabItem';
+import ParserOpenRPC from "@site/src/components/ParserOpenRPC"
+import { NETWORK_NAMES } from "@site/src/plugins/plugin-json-rpc"
 
-import Description from "/services/reference/_partials/_eth_call-description.mdx";
-
-<Description />
-
-## Parameters
-
-import Params from "./_eth_call-parameters.mdx";
-
-<Params />
-
-## Returns
-
-import Returns from "/services/reference/_partials/_eth_call-returns.mdx";
-
-<Returns />
-
-## Example
-
-import Example from "/services/reference/_partials/_eth_call-example.mdx";
-
-<Example />
-
-### Request
-
-import Request from "./_eth_call-request.mdx";
-
-<Request />
-
-### Response
-
-import Response from "/services/reference/_partials/_eth_call-response.mdx";
-
-<Response />
+<ParserOpenRPC
+  network={NETWORK_NAMES.linea}
+  method="eth_call"
```

Comment:
There is richer data in the schema than presented in the UI:
- pattern
- example
<img width="1103" alt="image" src="https://github.com/user-attachments/assets/ec513066-e942-4c1b-98f3-3df2fa9b27da">

Reply:
<img width="1258" alt="image" src="https://github.com/user-attachments/assets/616a4c69-bcd4-4125-8de9-53ecd433763e">